index.js

```diff
@@ -1,27 +1,74 @@
 const mongoose = require('mongoose');
 
-// Import of the model Recipe from './models/Recipe.model.js'
-const Recipe = require('./models/Recipe.model');
-// Import of the data from './data.json'
-const data = require('./data');
+const Recipe = require('./models/Recipe.model'); // Import of the model Recipe from './models/Recipe.model.js'
 
-const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
+const data = require('./data'); // Import of the data from './data.json'
+
+
+
+const MONGODB_URI = 'mongodb://localhost:27017/recipe-index';
 
 // Connection to the database "recipe-app"
 mongoose
+
   .connect(MONGODB_URI, {
     useCreateIndex: true,
     useNewUrlParser: true,
     useUnifiedTopology: true
   })
+
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
     // Before adding any documents to the database, let's delete all previous entries
-    return self.connection.dropDatabase();
+    return self.connection.dropDatabase(); // auto delete everything to avoid doublons
   })
+
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+
+    Recipe.create({
+      title: "Tortilla",
```

Comment:
You could save the object in a variable before the promise and the just insert the variable here. Usually helps with readability :)